api/openapi-spec/swagger.json

```diff
@@ -17473,7 +17473,7 @@
       "type": "string"
      },
      "fsFreezeStatus": {
-      "description": "FSFreezeStatus is the state of the fs of the guest it can be either frozen or thawed",
+      "description": "FSFreezeStatus reflects whether a freeze or thaw command was executed, not the actual filesystem state.",
```

Comment:
I agree that the description should change since as you stated it doesnt represent the actual fs state.
If you change this maybe its worth adding that FSFreezeStatus will only be frozen or wont exist.. so maybe no need to add the "or thaw" and maybe add "not the actual *guest* filesystem state"